experimental/dds/tree2/src/class-tree/README.md

```diff
@@ -38,12 +38,30 @@ Private data can still be stored using `#` private fields, or via weak keyed map
 Even regular private and protected fields can be used in the implementation and casts away from the type returned by the factory,
 though doing this risks name collisions with user added members.
 
-Recursive types are still somewhat sketchy.
-
 Comparing trees to object literals (for example in tests), will require a dedicated tree comparison function and/or comparing to unhydrated nodes (and implementing more APIs for them) instead of plain literals.
 
 Adding custom constructors to the schema classes is likely to break them, though static builders (like "create") can be added just fine.
 
+### Insertable content
+
+Currently the type allowed within InsertableContent comes from `InsertableTypedNode`.
+This type includes `NodeBuilderData<T>`, which directly comes from the constructor for the class.
```

Comment:
Nit: the phrasing here is a bit ambiguous. It would be good to explicitly note whose constructor we're referring to here.